src/Host.cpp

```diff
@@ -369,6 +369,36 @@ void Host::resetProfile()
     qDebug()<<"resetProfile() DONE";
 }
 
+// Saves profile to disk - does not save items dirty in the editor, however.
+// takes a directory to save in or an empty string for the default location
+// as well as a boolean whenever to sync the modules or not
+// returns true+filepath if successful or false+error message otherwise
+std::tuple<bool, QString, QString> Host::saveProfile(const QString& saveLocation, bool syncModules)
+{
+    QString directory_xml;
+    if (saveLocation.isEmpty()) {
+        directory_xml = QDir::homePath() + QLatin1String("/.config/mudlet/profiles/") + getName() + QLatin1String("/current");
+    } else {
+        directory_xml = saveLocation;
+    }
+
+    QString filename_xml = directory_xml + "/" + QDateTime::currentDateTime().toString("dd-MM-yyyy#hh-mm-ss") + ".xml";
```

Comment:
Is there any reason not to switch to the more sensible (IMHO) "yyyy-MM-dd#hh-mm-ss" date-time format (which sorts by name into chronological order ?) - the tweak to handle both types for the profile history list in the connection profiles dialogue is already in place (see: `(void) dlgConnectionProfiles::slot_item_clicked(QListWidgetItem *)`! :grinning: